### PR TITLE
[Mercure] integration with Symfony cli and various improvements

### PR DESCRIPTION
**Pull request by dunglas**
Sync the docs with the current best practices and document the Mercure support in Symfony CLI.

### CODE REVIEW

**Comment by dunglas** [comment]
The error is a false positive ("apps" is part of a link).

File: mercure.rst [hunk 1]
```diff
@@ -693,6 +659,51 @@ Enable the panel in your configuration, as follows:
...
+                'http://example.com/books/1',
+                json_encode(['status' => 'OutOfStock'])
+            );
+
+            // Sync, or async (Doctrine, RabbitMQ, Kafka...)
+            $bus->dispatch($update);
+
+            return new Response('published!');
+        }
+    }
```
Comment (azjezz) [inline mercure.rst:704]: maybe we can remove this entirely?

IIRC, the message handler is deprecated, and is not registered by default.

Comment (dunglas) [reply mercure.rst:704]: Actually, we didn't deprecate the handler (https://github.com/symfony/mercure/blob/main/src/Messenger/UpdateHandler.php), so we should keep this section.

File: mercure.rst [hunk 2]
```diff
@@ -211,7 +210,7 @@ as patterns:
 .. code-block:: javascript
 
     // URL is a built-in JavaScript class to manipulate URLs
-    const url = new URL('http://localhost:3000/.well-known/mercure');
+    const url = new URL('/.well-known/mercure');
```
Comment (MaximePinot) [inline mercure.rst]: Is this working? :thinking: 

From the [`URL` documentation on MDN](https://developer.mozilla.org/en-US/docs/Web/API/URL/URL#parameters) :

> If `url` is a relative URL, `base` is required, and will be used as the base URL. If `url` is an absolute URL, a given base will be ignored.

Output from Chromium's console : 
```
> const url = new URL('/.well-known/mercure');
VM89:1 Uncaught TypeError: Failed to construct 'URL': Invalid URL
    at <anonymous>:1:14
```

Comment (dunglas) [reply mercure.rst]: Good catch, window.origin must be passed as 2nd parameter, I'll fix this.

**Review by OskarStark — APPROVED** [review]

**Comment by OskarStark** [comment]
Thanks Kévin.

**Comment by OskarStark** [comment]
I fixed the build in https://github.com/symfony/symfony-docs/commit/5b0869f1b10f961dc1f4cf7159a9d205f0098cff

**Pull request merged**